server/middleware/users.test.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express';
+import { NextFunction } from 'express-async-router';
+
+import { removeMarkup } from './users';
+
+
+describe('removeMarkup middleware', () => {
+  it('should remove markup from request body fields', () => {
```

Comment:
what do you mean by converting to HTML entities?

Reply:
I'm just thinking of things like Twitter bios... people often put HTML tags in them as a form of expression. Should be fine to leave them as long as the chevrons are converted to `&lt;` and `&gt;`, right?

Reply:
ah yeah good callout.  is a find-and-replace-all of `<` and `>` with lt/gt respectively enough to prevent xss? heh

Reply:
Not sure, tbh... I'd probably also escape `&`, `'`, and `"` though a library specific to this would probably be the safer option.

[he](https://www.npmjs.com/package/he) seems to be extremely popular (15m downloads weekly), but hasn't had any changes since 2018.
[sanitize-html](https://www.npmjs.com/package/sanitize-html) seems to have more active development, but only 1.3m weekly downloads.

Reply:
@MatthewBozin thoughts on using an alternative like this? also let us know if you're super busy with final interview stuff, we can get someone to take this over the finish line

Reply:
It looks like sanitize-html's latest build is failing, so he is looking like a better option, plus it seems to have a better set of methods for what we want. I'll look into bringing it in and creating some tests for it